core/cautils/scaninfo.go

```diff
@@ -94,7 +94,8 @@ const (
 )
 
 type PolicyIdentifier struct {
-	Name        string                        // policy name e.g. nsa,mitre,c-0012
+	ID          string                        // policy ID e.g. c-0012 - relevant only to kind=control
```

Comment:
This is the identifier. It does not make sense to have a name and ID and to add a comment regarding the types of identifiers.
You either create new structs for identifying controls/frameworks or that you change the `name` to `ID`/`identifier`

Reply:
Done